code/aind_smartspim_quantification/utils/generate_ccf_cell_count.py

```diff
@@ -171,13 +167,21 @@ def generate_25_um_ccf_cells(params: dict, micron_res: int = 25):
                 "source": {
                     "url": f"precomputed://{params['ccf_cells_precomputed']['output_path']}",
                     "transform": {
-                        "matrix": [[0, 0, 1, 0], [0, 1, 0, 0], [1, 0, 0, 0]],
+                        "matrix": [
+                            [1, 0, 0, 0, 0, 0],
+                            [0, 1, 0, 0, 0, 0],
```

Comment:
Is there a dynamic way we could generate this matrix for visualization based on the orientation of the acquisition? Or with this, it was already contemplated?

Reply:
Actually, I can just remove this since it is not being instantiated by ng_link and we have resolved it by rotating the image after registration